arm-servicebus/2014-09-01/swagger/servicebus.json

```diff
@@ -560,9 +560,343 @@
+            "schema": {
+              "$ref": "#/definitions/QueueResource"
+            }
+          }
+        }
+      },
+     "delete": {
+         "tags": [
+           "Queues"
+         ],
+         "operationId": "Queues_Delete",
+         "description": "Deletes a queue from the specified namespace and resource group.",
+         "externalDocs": {
+             "url": "https://msdn.microsoft.com/en-us/library/azure/mt639411.aspx"
+         },
+        "parameters": [
+          {
+              "name": "resourceGroupName",
+              "in": "path",
+              "required": true,
+              "type": "string",
+              "description": "The name of the resource group."
+          },
+          {
+              "name":"queueName",
+              "in": "path",
+              "required": true,
+              "type": "string",
+              "description": "The queue name."
+          },
+          {
+              "name": "namespaceName",
+              "in": "path",
+              "required": true,
+              "type": "string",
+              "description": "The namespace name."
+          },
+          {
+              "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+              "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {          
+            "200": {
+                "description": "Queue deleted."
+            }         
+        },
+        "x-ms-long-running-operation": false
+     },
+      "get": {
+          "tags": [
+            "Queues"
+          ],
+          "operationId": "Queues_Get",
+          "description": "Returns the description for the specified namespace.",
+          "externalDocs": {
+              "url": "https://msdn.microsoft.com/en-us/library/azure/mt639380.aspx"
+          },
+        "parameters": [
+          {
+              "name": "resourceGroupName",
+              "in": "path",
+              "required": true,
+              "type": "string",
+              "description": "The name of the resource group."
+          },
+          {
+              "name": "namespaceName",
+              "in": "path",
+              "required": true,
+              "type": "string",
+              "description": "The namespace name."
+          },
+          {
+              "name":"queueName",
+              "in": "path",
+              "required": true,
+              "type": "string",
+              "description": "The queue name."
+          },
+          {
+              "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+              "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+            "200": {
+                "description": "Successfully retrieved queue description.",
+                "schema": {
+                    "$ref": "#/definitions/QueueResource"
+                }
+            }
+        }
+      }
```

Comment:
List by  subscription and by ResourceGroup is required for registered resources and Queues is a proxyonly resource.
We do not support such API.